### PR TITLE
Adds a local volume sample for the Viewer CRD

### PR DESCRIPTION
**Pull request by swiftdiaries**
Also edited the README for viewer CRD usage

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/822)

### CODE REVIEW

**Comment by neuromage** [comment]
/lgtm

Thanks @swiftdiaries! Looks good to me. 
@vicaire or @IronPan can you please approve? Thanks.

File: backend/src/crd/samples/viewer/tensorboard-local-volume.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This example shows how to use a Kubernetes local persistent volume with Tensorboard
+# viewer. The definition below assumes you have a volume called
+# 'argo-volume' in the cluster. This persistent volume will be mounted to
```
Comment (jinchihe) [inline backend/src/crd/samples/viewer/tensorboard-local-volume.yaml]: should be "have a volume called '**argo-claim**' in the cluster" in the below definition?

Comment (neuromage) [reply backend/src/crd/samples/viewer/tensorboard-local-volume.yaml]: I believe it's correct, the volume should be called `argo-volume`. The PVC name is `argo-claim`.

**Pull request merged**